test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -221,26 +238,11 @@ def forward(self, x):
 
     # Export Torchvision models
 
-    def test_alexnet(self):
-        model = torchvision.models.alexnet(pretrained=True)
-        x = torch.randn(2, 3, 224, 224, requires_grad=True)
-        self.run_test(model, (x,))
-
-    def test_densenets(self):
-        model = torchvision.models.densenet121(pretrained=True)
-        x = torch.randn(2, 3, 224, 224, requires_grad=True)
-        self.run_test(model, (x,), rtol=1e-3, atol=1e-5)
-
     def test_googlenet(self):
```

Comment:
Maybe move the remaining model tests to their own files?

Reply:
to test_models.py?

Reply:
Yes. Please move model tests from this file to test_models.